back/src/companies/sirene/__tests__/redundancy.test.ts

```diff
@@ -0,0 +1,65 @@
+import { redundant } from "../redundancy";
+import { ErrorCode } from "../../../common/errors";
+
+test("redundant", async () => {
+  expect.assertions(10);
+
+  const fn1 = jest.fn();
+  const fn2 = jest.fn();
+
+  const fn = redundant<string>(fn1, fn2);
+
+  // test fn1 returns something
+  fn1.mockResolvedValueOnce("bar");
+
+  let response = await fn("foo");
+  expect(response).toEqual("bar");
+
+  // fn2 should not be called
+  expect(fn1).toHaveBeenCalled();
```

Comment:
Ça serait top de découper en plusieurs tests pour améliorer la lisibilité, du genre :

- `it should not call fallback when the first query succeeds`
- `it should call fallback when the first query fails`
- `it should throw an error when both queries fail`

Reply:
Je viens de séparer en plusieurs tests c'est mieux en effet !